### PR TITLE
Update next 5.0.1-canary.2 to avoid build errors

### PR DESCRIPTION
**Pull request by lusan**
Please check this [link](https://github.com/zeit/next.js/issues/3738) as it has already been discussed,
updating the file so that it is easier for others to start without build errors :)

### CODE REVIEW

**Comment by lusan** [comment]
@loganfsmyth Please review it

**Comment by timneutkens** [comment]
There is no need for Logan to look at this 😅, removing the message in the hope it also removed the notification.

File: examples/with-react-native-web/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "start": "next start"
   },
   "dependencies": {
-    "next": "latest",
+    "next": "5.0.1-canary.2",
```
Comment (timneutkens) [inline examples/with-react-native-web/package.json]: Next 5.1 will be out soon, lets keep it as is for now.

Comment (lusan) [reply examples/with-react-native-web/package.json]: Ok sure :)

**Pull request closed without merging**